runtime/rasdump/dmpsup.c

```diff
@@ -714,6 +735,9 @@ configureDumpAgents(J9JavaVM *vm)
 	}
 	j9mem_free_memory(agentOpts);
 
+	/* Free optionString memory at original address for a mappped tool dump */
+	if (optionStringOriginal) {j9mem_free_memory(optionStringOriginal);}
```

Comment:
- Same comment about comparison
- Please put the `j9mem_free_memory(optionStringOriginal)` statement on its own line.